src/hardware/vga_draw.cpp

```diff
@@ -739,6 +777,22 @@ static void VGA_ProcessSplit() {
 	vga.draw.address_line=0;
 }
 
+static uint16_t from_rgb32_to_565(const uint32_t rgb32)
```

Comment:
How about `from_rgb24_to_rgb565`?

RGB565 is packed to 16-bit, probably always, at least that seems to be a de-facto standard and is therefore implied.

RGB32 is confusing to me because it implies the colour definition contains 32 significant bits? Perhaps RGB24 would be better?

But then, the 24-bit values are stored as packed 32-bit integers in memory... whereas the RGB565 are stored as packed 16-bit integers, I assume. So we have some conflation of pixel formats, significant number of bits, and actual memory layout encoded in these names... or maybe I'm just not used to seeing these names are people generally know what they refer to? (I don't and I was slightly confused and had to read the actual conversion code).

Reply:
Actually, I found this now:
http://www.barth-dev.de/about-rgb565-and-how-to-convert-into-it/

And I like the RGB888 and RGB565 names the article uses. That makes everything consistent and clear.